docs/Troubleshooting.md

```diff
@@ -24,6 +28,33 @@ $ bundle exec spring stop
 
 Run shopify_app generator again.
 
+## Rails
+
+### Known issues with Rails `v6.1`
+
+If you recently upgraded your application's `Rails::Application` configuration to load the default configuration for Rails `v6.1`, then you will need to update the following `cookies_same_site_protection` ActionDispatch configuration.
+
+```diff
+# config/application.rb
+
+require_relative 'boot'
+
+require 'rails/all'
+
+Bundler.require(*Rails.groups)
+
+module AppName
+  class Application < Rails::Application
++    config.load_defaults 6.1
+
++    config.action_dispatch.cookies_same_site_protection = :none
+    ...
+  end
+end
+```
+
+As of Rails `v6.1`, the same-site cookie protection setting defaults  to `Lax`. This does not allow an embedded app to make cross-domain requests in the Shopify Admin.
```

Comment:
Will the solution from `Upgrading` fix this? If so, we could also suggest that as a fix.

Reply:
That's a good point. I've added a line just below this that suggests an alternative to this is to upgrade to `v17.2.0` of the shopify_app gem. Unfortunately there's limited developer support for configuring the `SameSiteCookieMiddleware` at this time.